action.yml

```diff
@@ -16,10 +16,17 @@ inputs:
     default: |
       MATLAB
       Parallel_Computing_Toolbox
+  use-cache:
```

Comment:
It seems like this is just `cache` in a number of GitHub's own `setup-*` actions (`setup-dotnet`,`setup-go`,`setup-java` (not a boolean),`setup-node` (not a boolean)). Better to follow their convention?

Reply:
Changed to `cache` for now